web/packages/teleterm/src/services/logger/loggerService.ts

```diff
@@ -89,9 +89,10 @@ export function createFileLoggerService(
     transports: [
       new transports.File({
         maxsize: 4194304, // 4 MB - max size of a single file
-        maxFiles: 5,
+        maxFiles: opts.dev ? 5 : 3,
```

Comment:
IMO keeping 3 files for dev version should be fine.

Reply:
Yeah, but there was an instance or two where I grepped for an error that was happening for a long time (and we realized only after x weeks that it's there). In those cases, it's just better to have more logs.